backend/src/index.js

```diff
@@ -0,0 +1,30 @@
+const { mongoose } = require('mongoose');
+const express = require('express');
+const cors = require('cors');
+
+const app = express();
+
+app.use(express.json());
+
+// Middlewares
+app.use(cors());
+
+// routes
+
+app.use('/', require('./Routes/routes'));
+app.use('/todos', require('./Routes/routes'));
+app.use('/todos/:id', require('./Routes/routes'));
+
+// db connection
+mongoose
+  .connect('mongodb+srv://vladimir:R8QqnGKLQrEW1F8v@cluster0.8wijq.mongodb.net/todo_app?retryWrites=true&w=majority', {
```

Comment:
remove credential from the file,  credentials should be saved in .env file which will not be provided on github. This way everybody can have a access to your database.